src/ostorlab/cli/ci_scan/__init__.py

```diff
@@ -1,3 +1,5 @@
 """Module for the root command ci_scan"""
 from ostorlab.cli.ci_scan import run
 from ostorlab.cli.ci_scan.ci_scan import ci_scan
+
+__all__ = [run, ci_scan()]
```

Comment:
```suggestion
__all__ = [run, ci_scan]
```

Reply:
allright! That's why pytest wasn't running correctly.